modules/machineset-gcp-confidential-vm.adoc

```diff
@@ -13,10 +13,12 @@ endif::[]
 
 By editing the machine set YAML file, you can configure the Confidential VM options that a machine set uses for machines that it deploys.
 
-For more information about Confidential Compute features, functionality, and compatibility, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/confidential-vm/docs/about-cvm[Confidential VM].
+For more information about Confidential Compute features, functions, and compatibility, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/confidential-vm/docs/about-cvm[Confidential VM].
```

Comment:
The area linked to is actually named "Confidential Computing concepts," and the original sentence seems to have subject-verb disagreement ("for information about Confidential Compute features, functions, ..." "see [...] documentation about Confidential VM"). The actual section titled "Confidential VM" is here: https://cloud.google.com/confidential-computing/confidential-vm/docs/about-cvm#confidential-vm

Would this work better with the full page name or by creating subject-verb agreement and linking to that section?

Reply:
Thanks - I think the link has changed since this line was originally included in 4.13, and I should have checked it! The "Confidential Compute" in the subject is a mistake, fixing to say "Confidential VM".